src/main/tables/108_user_instant_launches.sql

```diff
@@ -0,0 +1,15 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS user_instant_launches (
+    id UUID UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
+
+    user_id UUID UNIQUE NOT NULL,
```

Comment:
Since we have a version on default instant launch settings, we might need a version of user-specific instant launch settings as well. Also, a uniqueness constraint on the `user_id` and `version` columns might be useful here.

Reply:
Added, thanks.